### PR TITLE
Remove elasticsearch functions moving to library

### PR DESCRIPTION
**Pull request by HeJones**
### What

Remove following elasticsearch functions and move to the elasticsearch library

- create index

- delete index

- add document

Update elasticsearch library to v2.0.1

### How to review

Ensure that the test still pass, and an upload process creates indexes in elasticsearch

### Who can review

Anyone other than me

### CODE REVIEW

File: elasticsearch/api.go [hunk 1]
```diff
@@ -72,73 +66,19 @@ func (api *API) AddDimensionOption(ctx context.Context, instanceID, dimension st
 		return 0, errors.New("missing dimension option code")
 	}
 
-	path := api.url + "/" + instanceID + "_" + dimension + "/dimension_option/" + dimensionOption.Code
+	indexName := instanceID + "_" + dimension
+	indexType := "dimension_option"
```
Comment (janderson2) [inline elasticsearch/api.go]: I would suggest putting this as a `const` at the top of the file with a descriptive name like indexTypeDimensionOption or something. That way it is available to all functions if you need to reuse it and it is clear where to change it if the value ever needs to be changed.

**Review by janderson2 — COMMENTED** [review]
Looking good overall. Much cleaner looking with the new library being used. One very minor comment.

File: elasticsearch/api.go (hunk 1 above)
Comment (HeJones) [reply elasticsearch/api.go]: Updated to a const in the latest commit

**Review by CarlHembrough — APPROVED** [review]
Good work 👍

**Pull request merged**